<launch>

    <arg name="odom_frame_id"   default="odom"/>
    <arg name="base_frame_id"   default="base_footprint"/>
    <arg name="global_frame_id" default="map"/>
    <arg name="odom_topic" default="odom" />
    <arg name="laser_topic" default="/hsrb/base_scan" />
    <arg name="map_topic" default="map" />

    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    	<rosparam file="$(find hsr_ros)/param/move_base_params.yaml" command="load" />
    	<rosparam file="$(find hsr_ros)/param/base_local_planner_params.yaml" command="load" />
    	<rosparam file="$(find hsr_ros)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    	<rosparam file="$(find hsr_ros)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    	<rosparam file="$(find hsr_ros)/param/local_costmap_params.yaml" command="load" />
    	<rosparam file="$(find hsr_ros)/param/global_costmap_params.yaml" command="load" />

        <!-- remap from="cmd_vel" to="navigation_velocity_smoother/raw_cmd_vel"/ -->
        <remap from="cmd_vel" to="/hsrb/command_velocity"/>
        <remap from="odom" to="$(arg odom_topic)"/>
        <remap from="scan" to="$(arg laser_topic)"/>
        <remap from="map" to="$(arg map_topic)" />
    </node>

</launch>
